pymatgen/core/surface.py

```diff
@@ -515,8 +323,8 @@ def get_surface_sites(self, tag=False):
         for analysis involving broken bonds and for finding adsorption sites.
```

Comment:
What is "elemental system" in the docstring
```
        We use VoronoiNN to determine the coordination number of sites.
        Due to the pathological error resulting from some surface sites in the
        VoronoiNN, we assume any site that has this error is a surface
        site as well. This will work for elemental systems only for now.
        Useful for analysis involving broken bonds and for finding adsorption sites.
```

Reply:
systems consisting only of one type of element

Reply:
feel free to replace with "single-element system" for clarity

Reply:
Thanks will do. Would appreciate it if you could give me a hand with #3694. Tried my best to fix most `mypy` errors but still a few left. Thanks! 😃